features/step_definitions/pull_request_steps.rb

```diff
@@ -0,0 +1,3 @@
+Then(/^I see a new (.+?) pull request for the "(.+?)" branch in the "(.+?)" repo in my browser$/) do |domain, branch, repo|
+  expect(@last_run_result.out).to eql "#{@tool} called with: #{pull_request_url domain, branch, repo}\n"
+end
```

Comment:
Everything in the specs is about something that comes from running a git town command. So I don't think this way of looking at it is very helpful here. This is about seeing a pull request in the browser. `run_steps` are for things when the user runs something on the command line ("when I run 'git foo'...").